Centaurus.Domain/Snapshot/SnapshotManager.cs

```diff
@@ -46,7 +46,8 @@ public static async Task ApplyInitUpdates(MessageEnvelope envelope)
                 Vault = initQuantum.Vault,
                 VaultSequence = initQuantum.VaultSequence,
                 Ledger = initQuantum.Ledger,
-                Pubkey = envelope.Signatures.First().Signer
+                Pubkey = envelope.Signatures.First().Signer,
```

Comment:
Why you take the first signer? Maybe this should be a `initQuantum.Vault`?

Reply:
I removed it. It used for user notifications. We don't need it for internal quanta.